.circleci/config.yml

```diff
@@ -67,18 +96,38 @@ jobs:
           paths:
             - html-scipyorg
 
+# Run benchmarks
+
+  run_benchmarks:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: ~/
+
+      - run:
+          <<: *debian
+
+      - run:
+          name: run asv
+          command: |
+            . venv/bin/activate
+            export PYTHONPATH=$PWD/build/testenv/lib/python3.7/site-packages
+            cd benchmarks
+            asv machine --machine CircleCI
+            asv --config asv.conf.json dev -m CircleCI --python=same --bench [-q]
```

Comment:
I don't understand this [-q] regular expression? It has been awhile since I worked with regular expressions, but doesn't this restrict to only test cases containing a `q` or a `-` somewhere in the name or parameters?

With `--bench [-q]` 41 benchmarks are found, but without this argument asv finds 234 benchmarks.

Reply:
I see above that this -q was probably intended to cause "quick" operation but it is instead being interpreted as a regular expression corresponding to the  ``--bench`` argument.

According to the ASV docs `asv dev` is already equivalent to `asv run --quick --show-stderr --python=same`, so I don't think `-q` needs to be specified to get quick operation.

Unfortunately, removing it is going to make running the benchmarks take much longer as many that didn't match the regular expression were being skipped.

Reply:
> I see above that this -q was probably intended to cause "quick" operation but it is instead being interpreted as a regular expression corresponding to the --bench argument.

Indeed, I assumed that this was intentional from comments above but I think I misinterpreted them. I agree it's not great to skip that many. I guess we'll see how long it takes to run all of them.

Maybe we'll need to somehow mark some benchmarks as slow so that they aren't run on CircleCI

Reply:
https://github.com/scipy/scipy/pull/12732 would give us some ideas for how to do this maybe